
# used to set control type

uint8 CLOSED_LOOP = 0
uint8 OPEN_LOOP = 1

uint8 DoF
uint8 Mode
